rust/noosphere-core/Cargo.toml

```diff
@@ -18,6 +18,10 @@ repository = "https://github.com/subconsciousnetwork/noosphere"
 homepage = "https://github.com/subconsciousnetwork/noosphere"
 readme = "README.md"
 
+[features]
+default = ["sentry"]
```

Comment:
We should probably make it opt-in, not opt-out. Folks will only want it if they happen to have a Sentry account. And, if they do want it, it's easy to add a feature to their dependency configuration:

```toml
noosphere-core = { ..., features = ["sentry"] }
```